packages/edit-site/src/components/global-styles/custom-css.js

```diff
@@ -1,130 +1,31 @@
 /**
  * WordPress dependencies
  */
-import { useState } from '@wordpress/element';
-import {
-	TextareaControl,
-	Panel,
-	PanelBody,
-	__experimentalVStack as VStack,
-	Tooltip,
-	Icon,
-} from '@wordpress/components';
-import { __ } from '@wordpress/i18n';
-import {
-	privateApis as blockEditorPrivateApis,
-	transformStyles,
-} from '@wordpress/block-editor';
-import { info } from '@wordpress/icons';
+import { privateApis as blockEditorPrivateApis } from '@wordpress/block-editor';
 
 /**
  * Internal dependencies
  */
 import { unlock } from '../../private-apis';
-import Subtitle from './subtitle';
 
-const { useGlobalStyle } = unlock( blockEditorPrivateApis );
-function CustomCSSControl( { blockName } ) {
-	// If blockName is defined, we are customizing CSS at the block level:
-	// styles.blocks.blockName.css
-	const block = !! blockName ? blockName : null;
-
-	const [ customCSS, setCustomCSS ] = useGlobalStyle( 'css', block );
-	const [ themeCSS ] = useGlobalStyle( 'css', block, 'base' );
-	const [ cssError, setCSSError ] = useState( null );
-	const ignoreThemeCustomCSS = '/* IgnoreThemeCustomCSS */';
-
-	// If there is custom css from theme.json show it in the edit box
-	// so the user can selectively overwrite it, rather than have the user CSS
-	// completely overwrite the theme CSS by default.
-	const themeCustomCSS =
-		! customCSS && themeCSS
-			? `/* ${ __(
-					'Theme Custom CSS start'
-			  ) } */\n${ themeCSS }\n/* ${ __( 'Theme Custom CSS end' ) } */`
-			: undefined;
```

Comment:
It would be worth keeping this bit of logic for theme-level CSS, as it helps make clear which styles will be preserved on reset when there are both theme.json and user-set styles.

Reply:
No, I meant the snippet removed from lines 40-45, that adds code comments above and below the theme CSS. If users want to add their own CSS while preserving the theme's, it may be helpful to have those comments delineating what belongs to the theme as opposed to what was user added.

Reply:
I'm happy to implement that but personally I don't like it that much as a user can still remove some parts of theme CSS and save the content and the comments will persist but the CSS will be different than the theme CSS.

Reply:
Hmm that's a valid point, and I suppose users can also add their own comments in there too. I'm not super invested in it either way, so it's not a blocker from my side.